src/bdd_2d.jl

```diff
@@ -0,0 +1,381 @@
+## Matlab Author: Rodrigo de Barros Vimieiro
+## Translated by Sonia Minseo Kim
+
+## 2-D Branchless Distance Driven Code
+using Plots: plot
```

Comment:
```suggestion
#using Plots: plot
```

Plots is an expensive dependency so we want to avoid it.  We can just comment it out.